pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -220,7 +220,7 @@ func (nc *nodeConfig) waitForNodeAnnotation(annotation string) error {
 	err := wait.Poll(retry.Interval, retry.Timeout, func() (bool, error) {
 		node, err := nc.k8sclientset.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
 		if err != nil {
-			return false, errors.Wrapf(err, "error getting node %s", nodeName)
```

Comment:
We should log these errors. The polls in the suite should need to be fixed as well, right?

Reply:
I think we would throw an error at the last, so we should be good.

Reply:
I think @sebsoto wants logs for the transient errors. It might be an useful indicator.